influunt-app/app/scripts/controllers/controladores/mapa.js

```diff
@@ -227,6 +253,171 @@ angular.module('influuntApp')
           .value();
       };
 
+      registerWatchers = function() {
+        pahoProvider.connect()
+          .then(function() {
+            pahoProvider.register(eventosDinamicos.ALARMES_FALHAS, alarmesEFalhasWatcher);
+            pahoProvider.register(eventosDinamicos.TROCA_PLANO, trocaPlanoWatcher);
+            pahoProvider.register(eventosDinamicos.STATUS_CONTROLADORES, statusControladoresWatcher);
+            pahoProvider.register(eventosDinamicos.CONTROLADOR_ONLINE, onlineOfflineWatcher);
+            pahoProvider.register(eventosDinamicos.CONTROLADOR_OFFLINE, onlineOfflineWatcher);
+          });
+      };
+
+      onlineOfflineWatcher = function(payload) {
+        var mensagem = JSON.parse(payload);
+        var controlador = _.find($scope.listaControladores, {id: mensagem.idControlador});
+
+        if (!controlador) {
+          console.log('controlador', mensagem.idControlador, 'não existe.');
```

Comment:
Sobrou um `console.log()` aqui. Mas se ele não for removido, recomendo colocar um espaço depois de `controlador` e outro antes de `não existe`, para o ID do controlador não ficar grudado nas palavras.